### PR TITLE
Update duration remaining to seconds

### PR DESCRIPTION
**Pull request by kmadsen**
## Description

Duration remaining [has been in milliseconds](https://github.com/mapbox/mapbox-navigation-android/pull/2664) when it should have been in seconds to be consistent.

- [x] I have added any issue links
- [ ] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [ ] I have added the appropriate milestone and project boards

### Implementation

Did an audit of the timestamps and found that most duration is being calculated with Double seconds. https://github.com/mapbox/mapbox-navigation-android/issues/2667 (@Guardiola31337 also told me [here](https://github.com/mapbox/mapbox-navigation-android/pull/2664#issuecomment-605279689))
## Screenshots or Gifs

Please include all the media files to give some context about what's being implemented or fixed. It's not mandatory to upload screenshots or gifs, but for most of the cases it becomes really handy to get into the scope of the feature / bug being fixed and also it's REALLY useful for UI related PRs screenshot gif

## Testing

Please describe the manual tests that you ran to verify your changes

- [x] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [ ] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR

### CODE REVIEW

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt [hunk 1]
```diff
@@ -63,7 +63,7 @@ private data class DynamicallyUpdatedRouteValues(
     val timeSinceLastReroute: AtomicInteger = AtomicInteger(0),
     var sessionId: String = TelemetryUtils.obtainUniversalUniqueIdentifier(),
     val distanceCompleted: AtomicReference<Float> = AtomicReference(0f),
-    val durationRemaining: AtomicLong = AtomicLong(0),
+    val durationRemaining: AtomicInteger = AtomicInteger(0),
```
Comment (kmadsen) [inline libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt]: didn't see the AtomicDouble. also don't feel like we really need Atomic here 🤔

**Comment by kmadsen** [comment]
I pulled master and received the same CI errors so 🤷‍♂

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt (hunk 1 above)
Comment (Guardiola31337) [reply libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt]: > also don't feel like we really need Atomic here 🤔 .

@olegzil could you clarify why this was designed using Atomic?

Comment (olegzil) [reply libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt]: dynamicValues.durationRemaining is being set from multiple places in the code. It is also being read in response to telemetry events.  As long as that's the case you have to guard against the dreaded ConcurrentModificationException, which translates into an Atomic.

**Pull request merged**